modules/carousel/jetpack-carousel.php

```diff
@@ -115,7 +134,7 @@ function display_bail_message( $output= '' ) {
 		return $output;
 	}
 
-	function enqueue_assets( $output ) {
+	function check_and_enqueue_assets( $output ) {
```

Comment:
It’s hard for me to know the core difference between `check_and_enqueue_assets` and `enqueue_assets`. What checks are those? In what cases we do/don’t need them?